src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -529,4 +529,21 @@
   <data name="PreferIsKindFix" xml:space="preserve">
     <value>Use 'IsKind' instead of 'Kind'</value>
   </data>
+  <data name="MissingSourceGeneratorAttributeDescription" xml:space="preserve">
+    <value>'{0}' implements 'ISourceGenerator' but does not have the 'Generator' attribute.</value>
```

Comment:
It's been some time but I think @Youssef1313 told me we should not be using placeholders in descriptions.

Reply:
@Evangelink Yes this is correct. Good catch.

Reply:
Is the preference to use "A type implements 'ISourceGenerator'"... ?

Reply:
@ryzngard Yes I think the wording looks good.